### PR TITLE
Measurement values units

### PR DESCRIPTION
**Pull request by edetec**
* Add measure unit support 
* Separate the measure component unit tests

### CODE REVIEW

**Review by lvoliveira — APPROVED** [review]

File: src/ng-xform/checkbox-field/checkbox-field.component.ts [hunk 1]
```diff
@@ -19,7 +19,8 @@ export class CheckboxFieldComponent extends BaseDynamicFieldComponent<CheckboxFi
 
   ngAfterContentInit() {
     if (this.control.value === null) {
-      this.control.setValue(false);
+      // this setTimeout is to avoid ERROR TypeError
```
Comment (igortg) [inline src/ng-xform/checkbox-field/checkbox-field.component.ts:22]: Add a `// TODO: Remove after RFDAP-593`

File: src/ng-xform/field-components/base-dynamic-field.component.ts [hunk 2]
```diff
@@ -19,6 +19,7 @@ export class BaseDynamicFieldComponent<T extends DynamicField> implements OnInit
 
   ngOnInit() {
     this.control = <FormControl>this.form.controls[this.field.key];
+    // console.log('control', this.control, this.field,  this.form.controls);
```
Comment (igortg) [inline src/ng-xform/field-components/base-dynamic-field.component.ts]: avoid commit commented code

File: src/types/js-quantities/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
+// Type definitions for JS-quantities 1.6
```
Comment (igortg) [inline src/types/js-quantities/index.d.ts:3]: Just add a comment that the version available on the DefinitelyTyped is not up to date, so we are adding a custom `.d.ts` for js-quantiles

File: src/types/js-quantities/README.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Installation
```
Comment (igortg) [inline src/types/js-quantities/README.md]: Do we really need this file? Remove it and add the `js-quantiles` module must be installed by the user

**Review by igortg — COMMENTED** [review]
No further review needed

File: src/ng-xform/checkbox-field/checkbox-field.component.ts (hunk 1 above)
Comment (edetec) [reply src/ng-xform/checkbox-field/checkbox-field.component.ts:22]: OK

File: src/ng-xform/field-components/base-dynamic-field.component.ts (hunk 2 above)
Comment (edetec) [reply src/ng-xform/field-components/base-dynamic-field.component.ts]: ops

File: src/types/js-quantities/index.d.ts (hunk 3 above)
Comment (edetec) [reply src/types/js-quantities/index.d.ts:3]: OK

File: src/types/js-quantities/README.md (hunk 4 above)
Comment (edetec) [reply src/types/js-quantities/README.md]: I'll remove it.

**Pull request merged**